Core/src/org/sleuthkit/autopsy/python/JythonModuleLoader.java

```diff
@@ -91,7 +101,9 @@
                             if (line.startsWith("class ") && filter.accept(line)) { //NON-NLS
                                 String className = line.substring(6, line.indexOf("("));
                                 try {
-                                    objects.add(createObjectFromScript(interpreter, script, className, interfaceClass));
+                                    if (interpreter != null) {
```

Comment:
This check adds a level of cyclomatic complexity that does not really do anything, given that we have already returned if there was an exception creating the interpreter, right?